tests/suites/test_suite_base64.function

```diff
@@ -105,7 +105,7 @@ void base64_decode_hex_src( data_t * src, char * dst_ref, int result )
     }
 
 exit:
-    ;;
+    ;
```

Comment:
The empty `exit` label can be removed completely, the test framework will add it automatically, but this doesn't hurt anything.